kod/object/item.kod

```diff
@@ -220,46 +220,76 @@ messages:
       return;
    }
 
-   CreateDesc(bShowall=FALSE)
+   CreateDesc(bShowAll=FALSE)
+   "Creates the item's full description text by combining the base description "
```

Comment:
Always document the return value.  In this case you can simply change this to say "Returns the item's full description..."